ui/src/common/QualityInfo.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import Chip from '@material-ui/core/Chip'
+
+export const QualityInfo = (props) => {
+  let { suffix, bitRate } = props.record
+  suffix = suffix.toUpperCase()
+  let info = suffix
+  if (bitRate !== '') {
```

Comment:
I think I was not clear in my previous comment. Let me explain:
Lossless formats (FLAC, WAV, DSF, ....) they DO have the bitrate value, but there's no need to display it, as they are Lossless, so in terms of quality, it does not really matter the bitrate for these formats.

I suggest you keep a list (in the `consts.js` file) of Lossless formats, and check if `suffix` is in it. In that case, show the `bitRate`.

It helps if you have some FLAC music in your library, so you can see the results. I expect the QualityInfo for a FLAC file to be just `FLAC`, and not `FLAC 1003`, as it'd be with your current implementation:
<img width="149" alt="Screen Shot 2021-03-29 at 11 31 30 AM" src="https://user-images.githubusercontent.com/331353/112860981-5ddf5e00-9082-11eb-8174-1a270f5fc4a6.png">

Reply:
Can i make an array of Lossless formats and use includes method?

Reply:
Yes, I think that's the best option for now.